all/pom.xml

```diff
@@ -974,6 +974,10 @@
             <groupId>io.helidon.webclient</groupId>
             <artifactId>helidon-webclient-websocket</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I see different modules here and in bom pom. These should be aligned.

Reply:
Sorry, didn't get this one. You mean for `helidon-webclient-grpc`?